app/models/section.rb

```diff
@@ -0,0 +1,94 @@
+class Section
+  include ActiveModel::Model
+  include ActiveModel::Validations
+
+  attr_accessor :content_id, :base_path, :title, :summary, :body, :updated_at, :public_updated_at, :organisations, :update_type
+
+  validates :summary, presence: true
+  validates :title, presence: true
+  validates :body, presence: true, safe_html: true
+
+  def initialize(params)
+    @content_id = params.fetch(:content_id, SecureRandom.uuid)
+    @title = params.fetch(:title, nil)
+    @summary = params.fetch(:summary, nil)
+    @body = params.fetch(:body, nil)
+    @publication_state = params.fetch(:publication_state, nil)
+    @updated_at = params.fetch(:updated_at, nil)
+    @public_updated_at = params.fetch(:public_updated_at, nil)
+    @manual = params.fetch(:manual_content_id)
```

Comment:
Same question I asked about the `Manual` initializer.  Seems more pressing here as this method sets `@manual` with the value of :manual_content_id in the payload, but it probably means to set `@manual_content_id`.